### ISSUE TITLE
Styled components not working with Gatsby. Reproducible.

### ISSUE THREAD

**Issue by obedparla** [gatsbyjs/gatsby#20111]
## Description

It seems like Gatsby is not adding styled component's styles to the DOM. SC is correctly adding the classes to the DOM elements and you can even see them change dynamically, but nothing is styled correctly.

What is _very_ weird is that I got it to work by deleting by code and then copy-pasting it again (they were no more than 10 lines of code). However, it stopped working after refreshing and I couldn't get it to work ever again.

### Steps to reproduce

Install gatsby following the "Get started" steps: 

```gatsby new gatsby-site```

Then follow the steps on [Gatsby with Styled Components](https://www.gatsbyjs.org/packages/gatsby-plugin-styled-components/). 

Run the project:

```npm run develop```

Try to use a styled component > nothing happens.

You can see a very minimal repo with nothing more than the gatsby installation and styled-components added to it. 

[Repo here](https://github.com/obedparla/gatsby-styled-components-not-working)

### Expected result
Styled components should work as expected.

### Actual result

Styled components don't work.

### Environment

  System:
    OS: macOS 10.15.1
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 10.16.3 - ~/.asdf/installs/nodejs/10.16.3/bin/node
    Yarn: 1.17.3 - /usr/local/bin/yarn
    npm: 6.9.0 - ~/.asdf/installs/nodejs/10.16.3/bin/npm
  Languages:
    Python: 2.7.16 - /usr/local/bin/python
  Browsers:
    Chrome: 78.0.3904.108
    Firefox: 71.0
    Safari: 13.0.3
  npmPackages:
    gatsby: ^2.18.8 => 2.18.8 
    gatsby-image: ^2.2.34 => 2.2.34 
    gatsby-plugin-manifest: ^2.2.31 => 2.2.31 
    gatsby-plugin-offline: ^3.0.27 => 3.0.27 
    gatsby-plugin-react-helmet: ^3.1.16 => 3.1.16 
    gatsby-plugin-sharp: ^2.3.5 => 2.3.5 
    gatsby-plugin-styled-components: ^3.1.16 => 3.1.16 
    gatsby-source-filesystem: ^2.1.40 => 2.1.40 
    gatsby-transformer-sharp: ^2.3.7 => 2.3.7 
  npmGlobalPackages:
    gatsby: 2.18.5

**Comment by LekoArts** [comment]
I cloned your project and it works without problems:

![image](https://user-images.githubusercontent.com/16143594/70796126-c9c08900-1da1-11ea-977f-a57b9fbb2fb0.png)

This codesandbox also works:
https://codesandbox.io/s/styled-components-gatsby-minimal-25ys5

Can't seem to reproduce your problem.

**Comment by obedparla** [comment]
What! That's eery. I use SC on other projects without issues. I tried it with Gatsby starter, with my own project and with Gatsby Advanced Started (https://github.com/Vagr9K/gatsby-advanced-starter) and they all failed.

**Comment by ericnmurphy** [comment]
I cloned the project and I'm seeing the exact same problem, styled components don't work. Interestingly, it works in the build, just not in the development environment. I'm having this issue with my other Gatsby project as well.

  System:
    OS: Linux 5.3 Ubuntu 19.10 (Eoan Ermine)
    CPU: (4) x64 Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 10.15.2 - /usr/bin/node
    Yarn: 1.19.2 - /usr/bin/yarn
    npm: 6.13.4 - /usr/local/bin/npm
  Languages:
    Python: 2.7.17 - /usr/bin/python
  Browsers:
    Chrome: 79.0.3945.79
    Firefox: 71.0
  npmPackages:
    gatsby: ^2.18.8 => 2.18.8
    gatsby-image: ^2.2.34 => 2.2.34
    gatsby-plugin-manifest: ^2.2.31 => 2.2.31
    gatsby-plugin-offline: ^3.0.27 => 3.0.27
    gatsby-plugin-react-helmet: ^3.1.16 => 3.1.16
    gatsby-plugin-sharp: ^2.3.5 => 2.3.5
    gatsby-plugin-styled-components: ^3.1.16 => 3.1.16
    gatsby-source-filesystem: ^2.1.40 => 2.1.40
    gatsby-transformer-sharp: ^2.3.7 => 2.3.7
  npmGlobalPackages:
    gatsby-cli: 2.8.18

**Comment by obedparla** [comment]
I should've mentioned that I had the issue in development mode. I just checked the built page and it worked as expected.

**Comment by connorlindsey** [comment]
Have you checked out this page: [Styled Components for Gatsby](https://www.gatsbyjs.org/docs/styled-components/)

Looks like you also need to install these npm packages `styled-components babel-plugin-styled-components`

**Comment by ericnmurphy** [comment]
> Have you checked out this page: [Styled Components for Gatsby](https://www.gatsbyjs.org/docs/styled-components/)
> 
> Looks like you also need to install these npm packages `styled-components babel-plugin-styled-components`

They've already been installed in the repo the OP provided, and it doesn't help.

**Comment by connorlindsey** [comment]
> > Have you checked out this page: [Styled Components for Gatsby](https://www.gatsbyjs.org/docs/styled-components/)
> > Looks like you also need to install these npm packages `styled-components babel-plugin-styled-components`
> 
> They've already been installed in the repo the OP provided, and it doesn't help.

Woops, I missed that repo. I just cloned it and was also able to make a simple styled component with it.

**Comment by obedparla** [comment]
It is working as expected for you as well @connorlindsey ? Can you try refreshing the site a few times and making changes to the component? This feels weird, why would it work for 2 of us and not other 2?

**Comment by connorlindsey** [comment]
@obedparla It's working totally as expected on Firefox and Chrome. Here's the page-2 component with a styled component. 

```javascript
import React from "react"
import { Link } from "gatsby"
import styled from "styled-components"

import Layout from "../components/layout"
import SEO from "../components/seo"

const StyledDiv = styled.div`
  color: orange;
  background: blue;
  text-transform: uppercase;
`

const SecondPage = () => (
  <Layout>
    <SEO title="Page two" />
    <StyledDiv>
      <h1>Hi from the second page</h1>
      <p>Welcome to page 2</p>
    </StyledDiv>
    <Link to="/">Go back to the homepage</Link>
  </Layout>
)

export default SecondPage

```

**Comment by LekoArts** [comment]
Please also try upgrading your Node version to a newer version, use another browser or try it in incognito mode (to deactivate browser extensions).

**Comment by obedparla** [comment]
Ok wtf I located the issue after realizing that it worked on Chrome incognito but not on my normal window even after clearing the cache.

the issue was due to the Chrome extension "Loom". I have no idea why. I'm using SC in several other projects with a similar setup (Webpack, Babel, etc) with no issues.

Anyway, closing this issue. If you land here, verify whether SC runs on an incognito window and if it does, check your extensions ¯\_(ツ)_/¯
